DPGAnalysis/MuonSysAging/python/customiseForMuonAging.py

```diff
@@ -0,0 +1,42 @@
+import FWCore.ParameterSet.Config as cms
+
+import os
+
+from DPGAnalysis.MuonSysAging.DTChamberMasker_cff  import *
+from DPGAnalysis.MuonSysAging.RPCChamberMasker_cff import *
+from DPGAnalysis.MuonSysAging.CSCChamberMasker_cff import *
+from DPGAnalysis.MuonSysAging.GEMChamberMasker_cff import *
+from DPGAnalysis.MuonSysAging.ME0ChamberMasker_cff import *
+
+from Configuration.Eras.Modifier_phase2_muon_cff import phase2_muon
+
+def enableAgingAtReco(process):
+    "Enable muon masking/aging for digis entering muon reconstruction"    
+  
+
+    appendCSCChamberMaskerAtUnpacking(process)
+    appendDTChamberMaskerAtUnpacking(process) 
+    if phase2_muon.isChosen():
+        appendRPCChamberMaskerBeforeRecHits(process)
+        appendGEMChamberMaskerAtReco(process)    
+        appendME0ChamberMaskerAtReco(process)    
+    else :
+        appendRPCChamberMaskerAtUnpacking(process)
+
+    return process
+
+def appendAgingFileToGlobalTag(process):
+    "Append aging DB file to GT from DPGAnalysis/MuonSysAging/test for testing purposes"
+
+    process.GlobalTag.toGet = cms.VPSet(
+        cms.PSet(
+            record = cms.string('MuonSystemAgingRcd'),
+            tag = cms.string('MuonSystemAging_test'),
+            connect = cms.string('sqlite_file://' 
+                                 + os.environ['CMSSW_BASE'] 
+                                 + '/src/DPGAnalysis/MuonSysAging/test/MuonSystemAging.db')
+            )
```

Comment:
Hello @battibass
since I don't see the sqlite file in question among the files committed in this PR
I assume the pointer to MuonSystemAging.db is only a remnant
of local test
We discourage including in official sequences GT customizations.
If you confirmed, may you please remove this part from what you commit ?

Once the data formats were in 91x, the payloads with the masking you want can be added to the 2023 GT's of atocond, and consumed according to the same optional logic